README.md

```diff
@@ -58,23 +61,27 @@ Even though it is strongly preferred to rely on ECC P521 and SHA-512 where possi
+2. On completion of Bootz, EnrollZ service is notified to enroll a TPM on a specific control card and calls the device's `GetIakCert` API to get back an IAK and IDevID certs.
+   - During initial bootstrapping, an active control card must use its IDevID cert (part of switch's default SSL profile) for securing TLS connection. Once the device is provisioned with switch-owner-issued prod TLS cert in `certz` workflow, the device must always use that cert for all subsequent enrollz RPCs (such as `RotateOIakCert`).
    - Primary/active control card is also responsible for all RPCs directed to the secondary/standby control card. The mechanism of internal communication between the two control cards depends on the switch vendor and is out of scope of this doc.
-3. EnrollZ service uses the trust bundle/anchor obtained (in advance) from the switch vendor to verify signature over the IAK and IDevID certs, and ensure that the control card serial number in oIAK cert and oIDevID cert is the same.
-   - *Note: EnrollZ service must have access to the up-to-date switch vendor trust bundle/anchor needed to verify the signature over the IAK certificate. The mechanics of this workflow are out of scope of this doc, but the trust bundle could be retrieved from a trusted vendor portal on a scheduled basis.*
-4. EnrollZ service ensures that device identity fields in IAK cert match its expectations.
-5. EnrollZ service asks switch owner CA to issue an oIAK cert based on the IAK pub key and device identity fields.
-6. EnrollZ service obtains the oIAK cert from the CA and calls the device's `RotateOIakCert` API to persist the oIAK cert on the control card.
-7. The switch verifies that the IAK pub key in oIAK cert matches the one in IAK cert.
-8. The switch stores the oIAK cert in non-volatile memory and will present it in the TPM attestation workflow.
-9. EnrollZ service repeats the workflow for the second control card if one is available.
+   Since the switch owner cannot directly TLS authenticate standby card, it is the responsibility of an active card to do an auth handshake with the standby card based on the IDevID key pair/cert as described in [RMA Scenario](#rma-scenario).
+3. EnrollZ service uses the trust bundle/anchor obtained (in advance) from the switch vendor to verify signature over the IAK cert, and ensure that the control card serial number in IAK cert and IDevID cert is the same.
+   - *Note: EnrollZ service must have access to the up-to-date switch vendor trust bundle/anchor needed to verify the signature over the IAK and IDevID certificates. The mechanics of this workflow are out of scope of this doc, but the trust bundle could be retrieved from a trusted vendor portal on a scheduled basis.*
+4. EnrollZ service ensures that device identity fields in IAK and IDevID certs match its expectations.
+5. EnrollZ service asks switch owner CA to issue an oIAK and oIDevID certs based on the IAK and IDevID pub keys, respectively.
```

Comment:
will we issue a odevid for each controller at this point as part of enrollment? - i.e. for multiple controller cards do we  enroll both at the same time or one at a time?